docs/running-node.md

```diff
@@ -159,3 +236,8 @@ $ run_dcl_node -t observer -c <chain-id> <node-name>
 *   Get the list of nodes participating in the consensus for the last block:
     *   `dclcli tendermint-validator-set`.
     *   You can pass the additional value to get the result for a specific height: `dclcli tendermint-validator-set 100`.
+
+## Validator Node Maintenance
+
+*   `persistent_peers` field in `$HOME/.dcld/config/config.toml` should include the latest version of the validators list
```

Comment:
It would be great to have a simple script that can take `persistent_peers.txt` file as an input and update the node's config file (probably even restart the node optionally)
I think we can assume that `persistent_peers.txt` file is kept in the up-to-date state by DCL node admins (in a Slack channel for example).